src/main/fc/rc.c

```diff
@@ -245,8 +245,9 @@ static void calculateSetpointRate(int axis)
     } else
 #endif
     {
+        const uint8_t deadband = axis == FD_YAW ? rcControlsConfig()->yaw_deadband : rcControlsConfig()->deadband;
```

Comment:
We could make this more efficient by pre-calculating the divisor for non-yaw / yaw outside of the loop, and then only applying the correct one in each case.

Reply:
I moved calculating divisors in initRcProcessing()